api/v1/controllers/RSVPController.js

```diff
@@ -34,6 +34,16 @@ function createRSVP(req, res, next) {
       }
       res.body = rsvp.toJSON();
 
+      services.RegistrationService
```

Comment:
I'd rather have you save the attendee upon first fetch rather than fetching it twice from datastore.

Reply:
Was going to do that but we have to send the email after calling `RSVPService.createRSVP(...)`, since that method checks for duplicates, and throws an error if it find a duplicate.